app/scripts/directives/document-line-edit.html

```diff
@@ -5,10 +5,10 @@
       <input type='radio' ng-model="line.name" value='image' ng-checked="line.name === 'image'">Image
       <input type='radio' ng-model="line.name" value='build' ng-checked="line.name === 'build'">Build
     </div>
-    <input type='text' class="build-image-text" ng-class="markForDeletionClasses()" ng-if="selectedImageName == null" ng-model="line.value"/>
-    <input type='text' class="build-image-text" ng-class="markForDeletionClasses()" ng-if="selectedImageName != null" ng-init="line.value = (line.name === 'image') ? selectedImageName: line.value" ng-model="line.value"/>
+    <input type='text' class="build-image-text" ng-class="markForDeletionClasses(null)" ng-if="selectedImageName == null" ng-model="line.value"/>
```

Comment:
I think you should not pass null to these methods and instead check for undefined or lodash.empty or just rely on the falsiness of undefined in the methods that check these arguments.